### PR TITLE
RUBY-2229 Link to current support/contribution resources


### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

**Comment by p-mongo** [comment]
https://jira.mongodb.org/browse/RUBY-2229

**Comment by p-mongo** [comment]
Rendered: https://p-mongo.github.io/rendered-docs/driver/tutorial/support.html

https://p-mongo.github.io/rendered-docs/driver/tutorial/contribute.html

**Comment by techbelle** [comment]
On contribute.html the paragraph for Request Product Features has a link that is not rendering correctly on the page (I think you mean this to be a hyperlink)
Otherwise LGTM

File: README.md [hunk 1]
```diff
@@ -9,84 +12,79 @@ Documentation
...
-**PLEASE DO**
+**PLEASE DO:**
 
-* Provide as much information as possible about the issue.
-* Provide detailed steps for reproducing the issue.
-* Provide any applicable code snippets, stack traces and log data.
-* Specify version information for the driver and MongoDB.
+- Provide as much information as possible about the issue.
+- Provide detailed steps for reproducing the issue.
+- Provide any applicable code snippets, stack traces and log data.
```
Comment (egiurleo) [inline README.md:49]: How would you feel about specifically asking for "copy-pastable" code snippets (i.e. not screenshots)?

Comment (p-mongo) [reply README.md:49]: I am not against it but also I haven't seen anyone posting screenshots in our interaction media.

Comment (egiurleo) [reply README.md:49]: It happened once but I agree that it's not a common occurrence. 👍

**Pull request merged**